report/README.md

```diff
@@ -0,0 +1,49 @@
+# Test Suite Runner
+
+This folder contains scripts and docker definitions that can run the suite against an implementation.  The implementations are configured in _implementations.json_.
+
+## Add your implementation
+
+Setting up your application requires several steps.
+
+### Create a CLI
+
+To add your implementation, you'll need to start by creating a CLI application that takes the following parameters:
+
+- `--schema <PATH#POINTER>` - The relative URI to the schema, e.g. `../tests/draft2019-09/additionalItems.json#/0/schema`.
```

Comment:
The implication here is that the instance and schema are expected to be files in JSON format. Are any other file formats allowed or expected to be supported? Must these be files on the local filesystem, or are network-reachable URIs possibly going to be used as well?

Reply:
e.g. is this legal? `--schema https://json-schema.org/draft/2020-12/schema`

Reply:
These URIs point directly to the file and schema/instance inside of it.  The docker-compose is copying the test suite as it exists in this repo directly into the container.  Your CLI will need to find the file, open it, and navigate to the location indicated by the pointer fragment.  The resulting JSON data will be the schema or instance as appropriate.

Reply:
> e.g. is this legal? --schema https://json-schema.org/draft/2020-12/schema

I expect that could be accepted by your CLI, but the test suite won't submit that.  It'll only submit schemas that exist in the test suite.

Reply:
> These URIs point directly to the file and schema/instance inside of it. The docker-compose is copying the test suite as it exists in this repo directly into the container. Your CLI will need to find the file, open it, and navigate to the location indicated by the pointer fragment. 

Great, please add that to the README.

Reply:
I had intended to add a "how it works" section that would include this.